.github/workflows/build_docker_images.yml

```diff
@@ -72,3 +72,66 @@ jobs:
           context: ./docker/peft-gpu
           push: true
           tags: huggingface/peft-gpu
+
+  latest-cuda-bnb-source:
+    name: "Latest Peft GPU + bnb source [dev]"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cleanup disk
```

Comment:
I've never seen this before in a workflow file :O

Reply:
Yeah, I've copied that from transformers since I used to get some disk issues: https://github.com/huggingface/transformers/blob/main/.github/workflows/build-docker-images.yml#L25